plugin/nvim-treesitter.lua

```diff
@@ -0,0 +1,118 @@
+-- Last Change: 2022 Apr 16
+
+if vim.g.loaded_nvim_treesitter then
+  return
+end
+vim.g.loaded_nvim_treesitter = true
+
+-- setup modules
+require("nvim-treesitter").setup()
+
+local api = vim.api
+
+-- define autocommands
+local augroup = api.nvim_create_augroup("NvimTreesitter", {})
```

Comment:
augroup duplicated with the one in configs, this will fail loading treesitter for buffers :)

Reply:
should we share an augroup for nvim-treesitter globally or use one per autocmd group ? I've fixed this by change `NvimTreesitter` to `NvimTreesitterConfigs` in `configs.lua`

Reply:
That is a good question. I tried to 1:1 replicate the current structure (but missed a `clear = false` ;)).

But, yes, I think a global augroup would be cleaner. (I forgot that that `create_autocmd` also takes a string for the `group`, so the constant `create_augroup` isn't actually needed.)

Reply:
i think this would be a good solution overall :)